Neos.Neos/Classes/Controller/Module/Administration/UsersController.php

```diff
@@ -367,7 +380,19 @@ public function updateAccountAction(Account $account, array $roleIdentifiers, ar
         }
         $password = array_shift($password);
         if (strlen(trim(strval($password))) > 0) {
-            $this->userService->setUserPassword($user, $password);
+            try {
+                $this->userService->setUserPassword($user, $password);
+            } catch (\Exception $e) {
+                $this->addFlashMessage(
+                    // The shown message must be translated when throwing the actual exception.
+                    $e->getMessage(),
+                    $this->translator->translateById('userSettings.passwordUpdateFailed.title', [], null, null, 'Modules', 'Neos.Neos'),
+                    Message::SEVERITY_ERROR,
+                    [],
+                    1647335912
```

Comment:
The id should be different than the one in the `createAction` L196

Reply:
Done